### DISCUSSION TITLE
Default config file

### DISCUSSION THREAD

**Discussion by nathanielevan** [hrkfdn/ncspot#1339] [General]
Hi, I'm not sure if this has been covered elsewhere as I'm half sleepy right now (sorry!) but is there anywhere I can retrieve the default config file or have the program dump it to a certain file? Many thanks

**Comment by Bettehem** [comment]
Here you can see all the available config options and their default values:
https://github.com/hrkfdn/ncspot/blob/main/doc/users.md#configuration

**Reply by nathanielevan** [reply]
Right, I'm aware of the options available - just thought a default template would be a bit more convenient, but this will do. Thanks!
